v2h_templates.py

```diff
@@ -121,7 +121,11 @@
 <body>
 <main>
 <h1>Subway Validation Results</h1>
-<p>Total good metro networks: {good_cities} of {total_cities}.</p>
+<p><b>{good_cities}</b> of <b>{total_cities}</b> networks validated without errors.
+To make a network validate successfully please follow the
+<a href="https://wiki.openstreetmap.org/wiki/Metro_Mapping">metro mapping instructions</a>.
+After editing the OSM data check back to the updated validation results on the next day
```

Comment:
`on the next day` might be misleading, since there is a weekly schedule for automatic validation, and there is nothing about triggered validadion here (a `validate` url, mentioned in https://wiki.openstreetmap.org/wiki/Quality_assurance#subway-preprocessor)

Reply:
Changed text, added link to OSM wiki